src/main/java/com/google/sps/servlets/POJOCustomSearchNewsParser.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+class POJOCustomSearchNewsParser implements CustomSearchNewsParser {
```

Comment:
java naming conventions says even for acronyms you should use camelCase, e.g PojoCustomSearchNewsParser